pkg/volume/rbd/rbd_util.go

```diff
@@ -265,20 +274,33 @@ func (util *RBDUtil) AttachDisk(b rbdMounter) (string, error) {
 
 		// Currently, we don't acquire advisory lock on image, but for backward
 		// compatibility, we need to check if the image is being used by nodes running old kubelet.
-		found, rbdOutput, err := util.rbdStatus(&b)
-		if err != nil {
-			return "", fmt.Errorf("error: %v, rbd output: %v", err, rbdOutput)
-		}
-		if found {
-			glog.Infof("rbd image %s/%s is still being used ", b.Pool, b.Image)
-			return "", fmt.Errorf("rbd image %s/%s is still being used. rbd output: %s", b.Pool, b.Image, rbdOutput)
+		// osd_client_watch_timeout defaults to 30 minutes, in case of an ungraceful node restart
+		// a watcher could stays up to 30 minutes active as a result rbdStatus will block any attempts
```

Comment:
30 seconds

Reply:
30 minutes is based on real life tests. If it were 30 seconds then it would have not been an issue.